release_bot/releasebot.py

```diff
@@ -76,10 +76,18 @@ def find_open_release_issues(self):
                 for edge in reversed(edges):
                     cursor = edge['cursor']
                     match = re.match(r'(.+) release', edge['node']['title'].lower())
-                    if match and validate(match[1]) and \
-                            edge['node']['authorAssociation'] in ['MEMBER', 'OWNER', 'COLLABORATOR']:
-                        release_issues[match[1]] = edge['node']
-                        self.logger.info(f'Found new release issue with version: {match[1]}')
+                    if match:
+                        version = match[1].strip()
+                        if validate(version):
+                            if edge['node']['authorAssociation'] in ['MEMBER', 'OWNER', 'COLLABORATOR']:
+                                release_issues[version] = edge['node']
+                                self.logger.info(f'Found new release issue with version: {version}')
+                            else:
+                                self.logger.warning(
```

Comment:
One more question: so this will land only in a log -- if something goes wrong, we literally need to go to the openshift console a figure out what's wrong. Wouldn't it be better to also comment on the issue? WDYT?

Reply:
We'd need to implement the comment de-duplication mechanism, otherwise, we'd get a shitload of comments with the same warning. I know Github now merges them, but still.

Reply:
right, that's a good point; let's merge then and I link this conversation to the dedup issue